conferences/forms.py

```diff
@@ -25,10 +32,7 @@ class Meta:
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.fields['organization'].queryset = Organization.objects.filter(accepted=True)
-        bus_queryset = Bus.objects.find_with_free_places(Zosia.objects.find_active())
```

Comment:
I think old version was more readable. If you want to cut it from `__init__` then maybe move it to class method?